UITableView_pageSetting/ViewController.swift

```diff
@@ -8,12 +8,146 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    let idCell = "MyCell"
+    @IBOutlet weak var tableView: UITableView!
+    
+    var items: [String] = ["we", "heart", "Swift"]
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        tableView.dataSource = self
+        self.tableView.backgroundColor = UIColor.lightGray
+        self.tableView.sectionHeaderHeight = 10
     }
+    
+}
+
+struct Cell {
+    var title: String?
+}
 
 
+class CellApi {
+    static func getData() -> [[Cell]] {
+        let section1 = [Cell(title: "Авиарежим"),
+                        Cell(title: "Wi-Fi"),
+                        Cell(title: "Bluetooth"),
+                        Cell(title: "Сотовая связь"),
+                        Cell(title: "Режим модема"),
+                        Cell(title: "VPN")]
+
+        return [section1]
+    }
+}
+
+var allCellData = CellApi.getData()
+extension ViewController: UITableViewDataSource {
+    // MARK: Количество строк в секции
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if section == 0 {
+            return 6
+        } else if section == 1 {
+            return 4
+        } else if section == 2 {
+            return 4
+        } else {
+            return 0
+        }
+    }
+    
+    func label(width text: String) {
+        let _: UILabel = {
+         let n = UILabel()
+            n.textColor = UIColor.darkGray
+            n.textAlignment = .right
+            n.text = text
+            n.font = UIFont(name: "Montserrat", size: 12)
+         return n
+        }()
+    }
+    
+    
+    // MARK: клик на ячейку
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+        let title = allCellData[indexPath.section][indexPath.row].title
+        print("Выбрана ячейка \(title ??  "")")
+    }
+
+    // MARK: настройка ячеек
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
+        let cell = UITableViewCell(style: UITableViewCell.CellStyle.value1, reuseIdentifier: idCell)
+        if(indexPath.section == 0){
+            
+            if(indexPath.item == 0) {
+                cell.textLabel?.text = allCellData.first?[0].title
+                cell.imageView?.image = UIImage(named: "airplane")
+                cell.accessoryView = UISwitch()
+            } else if (indexPath.item == 1){
+                cell.textLabel?.text = allCellData.first?[1].title
+                cell.imageView?.image = UIImage(named: "wifi")
+                cell.accessoryType = UITableViewCell.AccessoryType.disclosureIndicator
+                cell.detailTextLabel?.text = "Не подключено"
```

Comment:
Лаконичней все это дело через оператор свитч оформить, да и вынести в структуру а в методе дергать свойство
```suggestion
           if(indexPath.item == 0) {
                cell.textLabel?.text = allCellData[indexPath.row].title
                cell.imageView?.image = UIImage(named: "airplane")
                cell.accessoryView = UISwitch()
            } else if (indexPath.item == 1){
                cell.textLabel?.text = allCellData[indexPath.row].title
                cell.imageView?.image = UIImage(named: "wifi")
                cell.accessoryType = UITableViewCell.AccessoryType.disclosureIndicator
                cell.detailTextLabel?.text = "Не подключено"
```